libsoftwarecontainer/component-test/softwarecontainerlib_componenttest.cpp

```diff
@@ -508,28 +522,29 @@ TEST_F(SoftwareContainerApp, TestUnixSocket) {
         sleep(1);
```

Comment:
Add comment explaining why there is a sleep here. If possible it would be nice to swap it for a `join(pid)` or something along those lines.